### PR TITLE
CFE-4160: Added classes and vars to cf-support

### PR DESCRIPTION
**Pull request by nickanderson**
Ticket: CFE-4160
Changelog: Title

### CODE REVIEW

File: misc/cf-support [hunk 1]
```diff
@@ -335,6 +335,10 @@ log_cmd "$BINDIR/cf-key -p $WORKDIR/ppkeys/localhost.pub"
 log_cmd "grep 'version =' $WORKDIR/inputs/promises.cf"
 log_cmd "$BINDIR/cf-key -s -n"
 log_cmd "$BINDIR/cf-check diagnose"
+$BINDIR/cf-promises --no-lock --show-classes --show-vars > "$tmpdir/cf-promises_--no-lock_--show-classes_--show-vars.txt" 2>&1
+$BINDIR/cf-agent --no-lock --file update.cf --show-evaluated-classes --show-evaluated-vars > "$tmpdir/cf-agent_--no-lock_--file_update.cf_--show-evaluated-classes_--show-evaluated-vars.txt" 2>&1
```
Comment (craigcomstock) [inline misc/cf-support:339]:
```suggestion
$BINDIR/cf-agent --no-lock --file update.cf --show-evaluated-classes --show-evaluated-vars > "$tmpdir/update-classes-and-vars.txt" 2>&1
```

**Review by craigcomstock — APPROVED** [review]
🏅

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/9388/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/9388/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-9388/

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/9400/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/9400/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-9400/

**Review by craigcomstock — APPROVED** [review]
ci build looks green enough. :+1:

**Review by olehermanse — APPROVED** [review]

**Pull request merged**